pkg/contexts/ocm/elements/artifactblob/helmblob/helmblob_test.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Open Component Model contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
the SPDX file headers aren't required anymore, since we have the `.reuse/dep5` file, which defines Copyright + License once and for all